### ISSUE TITLE
Pop OS - After installation only show external screen 

### ISSUE THREAD

**Issue by ceconelo** [AdnanHodzic/displaylink-debian#864]
I have two externals screen + notebook screen, but only one appears.

In my gnome-setting it disappeared the display mode option.

Can I helpe me?

--------------- Linux system info ----------------

Distro: Pop
Release: jammy
Kernel: 6.4.6-76060406-generic

---------------- DisplayLink info ----------------

Driver version: 1.14.1
DisplayLink service status: up and running
EVDI service version: 1.14.1

------------------ Graphics card -----------------

Vendor: nvidia
amdgpu
Subsystem: TU117M
Renoir
VGA: NVIDIA Corporation TU117M (rev a1)
Advanced Micro Devices, Inc. [AMD/ATI] Renoir (rev c6)
VGA (3D): 
X11 version: 21.1.4-2ubuntu1.7~22.04.1
X11 configs: /etc/X11/xorg.conf.d/20-displaylink.conf

-------------- DisplayLink xorg.conf -------------

File: /etc/X11/xorg.conf.d/20-displaylink.conf
Contents:
 Section "ServerLayout"
    Identifier "layout"
    Screen 0 "nvidia"
    Disable "intel"
EndSection

Section "Device"
    Identifier "intel"
    Driver "modesetting"
    Option "AccelMethod" "None"
EndSection

Section "Screen"
    Identifier "intel"
    Device "intel"
EndSection

Section "Device"
    Identifier "nvidia"
    Driver "nvidia"
    Option "ConstrainCursor" "off"
EndSection

Section "Screen"
    Identifier "nvidia"
    Device "nvidia"
    Option "AllowEmptyInitialConfiguration" "on"
    Option "IgnoreDisplayDevices" "CRT"
EndSection

-------------------- Monitors --------------------

Providers: number : 7
Provider 0: id: 0x1b7 cap: 0x1, Source Output crtcs: 4 outputs: 1 associated providers: 6 name:NVIDIA-0
Provider 1: id: 0x2e1 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 2: id: 0x2c0 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 3: id: 0x29f cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 4: id: 0x27e cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 5: id: 0x247 cap: 0x2, Sink Output crtcs: 1 outputs: 1 associated providers: 1 name:modesetting
Provider 6: id: 0x1f4 cap: 0x2, Sink Output crtcs: 4 outputs: 1 associated providers: 1 name:modesetting

-------------------------------------------------------------------

**Comment by tegaidogun** [comment]
I'm having the same issue.

**Comment by ceconelo** [comment]
> I'm having the same issue.

Hi, Do you founded any solution?

**Comment by tegaidogun** [comment]
Nope, moved back to Windows until I could afford a Gaming Laptop with DisplayPort 1.4 or higher Support.
